cli/pkg/fileutil/path.go

```diff
@@ -3,8 +3,33 @@ package fileutil
 import (
 	"os"
 	"path/filepath"
+	"strings"
 )
 
+func ToAbsDir(filePath string) (string, error) {
+	absPath, err := filepath.Abs(filePath)
+	if err != nil {
+		return "", err
+	}
+
+	return filepath.Dir(absPath), nil
+}
+
+func RelativeTo(path, relativeTo string) string {
+	if filepath.IsAbs(path) {
+		return path
+	}
+
+	return filepath.Join(relativeTo, path)
+}
+
+func LooksLikeFilePath(path string) bool {
+	return strings.HasPrefix(path, "./") ||
```

Comment:
Being a devil's advocate here, this method will fail for Windows if a user uses a full path, no?

We could have a transaction step as `C:/my-folder/my-test.yaml`

Reply:
You are right, I didn't think of that case. This is the current behaviour I think, so I'll keep it as is now. If the case happens, we'll figure out how to fix it :P